scripts/release.sh

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
+
+# This script is used to release a new version of the Testcontainers for Go library.
+# It creates a tag for the root module and for each module in the modules directory,
+# and then triggers the Go proxy to fetch the module.
+#
+# Usage: ./scripts/release.sh "v0.0.1"
+# where v0.0.1 is the tag to be created.
+#
+# It's possible to run the script in dry-run mode, which will print the commands that
+# would be executed, without actually executing them.
+#
+# Usage: DRY_RUN="true" ./scripts/release.sh "v0.0.1" 
+
+readonly DRY_RUN="${DRY_RUN:-false}"
```

Comment:
Should be the default `DRY_RUN` value `true`?

Reply:
Indeed. Let me change it now